---
title: research
layout: research
group: research
---

			<section id="main" class="wrapper">
				<div class="inner">
					<div class="row">
						<div class="3u 12u$(small)">
							<h3 id="showdiv1"><a href="#">Research</a></h3>
							<ul id="collapse" class="alt">
								<li id="showdiv2"><a href="#">Cytoskeletal dynamics and force generation in immune cells</a></li>
								<li id="showdiv3"><a href="#">Immune receptor dynamics </a>
								<li id="showdiv4"><a href="#">Cellular mechanosensing</a>
									<ul>
										<li id="showdiv5"><a href="#">Sensing substrate stiffness</a></li>
										<li id="showdiv6"><a href="#">Sensing substrate topography</a></li>
									</ul>
								</li>
								<li id="showdiv7"><a href="#">Cytoskeletal control of gene expression</a></li>
								<li id="showdiv8"><a href="#">Force transmission in actin networks</a></li>
							</ul>
						</div>
						<div id="research" class="9u$ 12u$(small)">
							<div id="div1" class="content">
								<p>The research in our lab broadly pursues two themes: (1) How do physical properties
									of the cell and its environment influence biochemical signaling (with particular
									emphasis on the immune response) (2) How do physical properties of crosslinked
									actin networks in the cell determine mesoscopic cell mechanics, cellular force generation
									and force response in cells. In order to answer these scientific questions we use state of
									the art imaging for the visualization of internal protein dynamics at high spatial and
									temporal resolution, traction force microscopy for measurements of cellular forces
									simultaneously with signaling, and magnetic tweezers to physically perturb the system.</p>
							</div>
							<div id="div2" class="content" style="display:none;">
								<h3>Cytoskeletal dynamics and force generation in immune cells</h3>
								<p>The interplay of physical forces and biochemical signaling pathways is critical in driving many biological phenomena including the immune response.
									T cells, which are central players in the immune response, bind to antigen presenting cells through specialized receptor-ligand interactions.
									Receptor engagement initiates actin dynamics which leads to deformation of the cell membrane and spatial patterning of signaling proteins into
									micro-clusters which are essential for signaling. We aim to understand the physical mechanisms governing contact formation between immune cells,
									signaling and spatial organization of signaling proteins.</p>
									<p><span class="image left"><img src="img/Figure_schematic_for_grant.jpg" alt=""/></span>A number of studies have shown that signaling by single and ensembles of
									T/B cell receptors are sensitive to externally applied forces. However, in the body, these forces have to be generated by the cell itself.
									The magnitude, spatial organization and temporal properties of internal cytoskeletal forces and whether they are sufficient to
									engage signaling were not known.  We showed that while T cells are weak force generators,
									the forces are sufficient to facilitate receptor activation, and that the forces are largely generated and maintained by the
									dynamics of the acto-myosin cytoskeleton [4]. These measuremements demonstrate that physiological level of
									forces are sufficient for biochemical activation of signaling receptors. </p>
									<p><span class="image right"><img src="img/MBOC_traction.png" alt=""/></span>Our recent work has shown a surprising and
										hitherto unknown role for microtubule filament dynamics in T cell force generation [5],
									and that the interaction between the actin and microtubule cytoskeletons is essential for optimum force balance
									in cells. Given the importance of dynamic force application in T cell activation, we have examined the statistics of
									force fluctuations during signaling. We found that cytoskeletal forces showed clear non-equilibrium character,
									indicative of active forcing, with a 1/f<sup>2</sup> power spectrum [6]. Using super-resolution imaging methods
								(instant Structured Illumination Microscopy), we found that these two spatially distinct regimes of force generation
								arise from different actin-based structures that have distinct molecular origins.</p>
								<p>Lab Members Involved: <a href="lab_folks.html#king">Altug Ozcelikkale</a>, <a href="lab_folks.html#saein">Kaustubh Wagh</a>, <a href="lab_folks.html#saein">David Garcia</a><br>
					      Collaborators: <a href="http://papoian.chem.umd.edu/">Papoian Lab, UMD Chemistry</a>, <a href="https://www.nibib.nih.gov/labs-at-nibib/section-high-resolution-optical-imaging-hroi">Shroff Lab, NIBIB/NIH</a><br>
					      Funding: <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1607645&HistoricalAwards=false">NSF Physics of Living Systems</a></p>
							</div>
							<div id="div3" class="content" style="display:none;">
								<h3>Immune receptor dynamics</h3>
								<p>B lymphocytes form a critical part of the immune response by generating antibody responses against
									invading pathogens and maintaining long lasting immunity. Binding of antigens to clonally specific
									B cell receptors (BCR) initiates B cell activation and the formation of signaling microclusters of
									antigen-engaged BCR. Despite its considerable importance, physical insight into the mechanisms of
									microcluster assembly is limited. Using a combination of TIRF, single molecule imaging and nano-fabrication
									techniques, we are studying how mesoscale reorganization of BCR is regulated by
									the dynamics of the actin cytoskeleton.</p>
								<p><span class="image right"><img src="img/Bcell_clustering_1_labels.png" width="60%" alt=""/></span>Cellular processes are often
									governed by molecular interactions or events, such as binding of cell surface receptors to extracellular ligand.
									These molecular scale events lead to cooperative behavior at the mesoscale such as aggregation into macromolecular
									complexes and emergence of larger scale cellular responses. How these different scales are linked is an open
									question in the field. Mobility of single cell surface molecules can provide insight into their biochemical state,
									the physical properties of their microenvironment and the forces acting on them. Our work has shown that lateral
									mobility of ligands is an important parameter regulating microcluster formation and signaling threshold [2].
									A recent goal of our lab has been to dissect the biochemical interactions and evolution of signaling states.
									We have established imaging techniques to quantify movement of single B cell receptors with high spatial and
									temporal resolution [10]. In collaboration with Simon Mochrie (Yale Univ.), we have used machine-learning
									methods to reveal the diffusive states populated by BCR and map these states onto BCR signaling states (Figure 2).
									We find that actin regulatory proteins (NWASP) modulate these diffusive and signaling states, indicating that
									actin based forces are important in driving receptor mobility, which in turn determines clustering and the
									biochemical response of B cells [11]. Our results are the first in the field to delineate multiple diffusive
									states of receptors and associate them with signaling properties. Together, these studies along with our
									ongoing collaborative work with Dr. Wenxia Song demonstrate that forces from actin polymerization directly
									receptor movement, challenging the prevailing hypothesis in the field that actin’s main role is to form diffusion
									barriers for BCR movement and underscores the importance of cytoskeletal forces in signaling. </p>
									<p>Lab Members Involved: <a href="lab_folks.html#king">Ivan Rey</a><br>
										Collaborators: <a href="http://papoian.chem.umd.edu/">Song Lab, UMD Cell Biology</a><br>
							      Funding: <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1563355&HistoricalAwards=false">NSF Biomechanics and Mechanobiology</a>
									</p>
							</div>
							<div id="div4" class="content" style="display:none;">
								<h3>Cytoskeletal mechanisms for sensing substrate stiffness</h3>
								<p>Physical properties of the extracellular environment can have profound influence on cell behavior in
									various physiological processes such as development, cell fate determination, the immune response and
									cancer metastases. For instance, substrate rigidity serves as a master regulator in the body to control
									cellular functions from cytoskeletal dynamics to gene expression. However, the mechanism by which cells
									sense and respond to the stiffness is an outstanding question in the field. Over the past four years,
									we have established a variety of experimental methods and collaborations with theorists to examine
									the cellular basis of mechanosensing. </p>
							</div>
							<div id="div5" class="content" style="display:none;">
								<h3>Sensing substrate stiffness</h3>
								<p><span class="image right"><img src="img/Bcell_clustering_1_labels.png" width="60%" alt=""/></span>
									A simple physical picture is obtained by considering the cell as an idealized stiffness-measuring device:
									during a measurement, the device exerts a local force on the substrate and reads out a local elastic modulus.
									How this device is realized by the structural and biochemical readout available to the cell and how it affects
									cell remains to be determined. While efforts have been made in this direction in various cellular contexts,
									how stiffness influences immune cell function has largely been unexplored until recently. We demonstrated that
									T cells are mechanosensitive, altering their signaling dynamics and forces
									generated in response to substrate stiffness [4]. We found that forces increase monotonically with stiffness
									of the external surface indicating that cells reorganize their cytoskeletal machinery adaptively. A model
									from active matter theory is able to explain this behavior based on cellular adaptation to substrate rigidity
									resulting from interplay between passive elasticity and active contractility.</p>

									<p><span class="image right"><img src="img/Bcell_clustering_1_labels.png" width="60%" alt=""/></span>The
										mechanisms underlying the origins of mechanosensitivity are not known, but likely involve adaptive
										changes in the organization and mechanical properties of the actin cytoskeleton with subsequent changes
										in cell compliance and shape. In support of this view, we showed that tumor associated fibroblast cells
										are sensitive to the stiffness of the substrate generating larger forces on stiffer substrates [8].
										This adaptive behavior required an actin-crosslinking protein (palladin) for the assembly of contractile
										myosin bundles for force generation. As palladin is implicated in some forms of pancreatic cancer, our
										studies suggest that actin crosslinkers and myosin motors tune the “mechanical state” of the cell and
										alterations in their function leads to aberrant behavior such as in cancer metastases. These results
										highlight the importance of cell mechanical properties in modulating cell function and re-iterate the
										connection between cytoskeletal network structure (regulated by actin crosslinkers) and dynamics in cells.
										Our results shed light on how stiffness of the tumor microenvironment regulates cancer cell shape and
										dynamics and will serve to constrain theoretical models of actin network assembly and force generation in cells.</p>
										<p>Lab Members Involved: <a href="lab_folks.html#king">Altug Ozcelikkale</a>,  <a href="lab_folks.html#king">Kaustubh Wagh</a><br>
						      	unding: <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1563355&HistoricalAwards=false">NSF Biomechanics and Mechanobiology</a>
							</div>
							<div id="div6" class="content" style="display:none;">
								<h3>Cytoskeletal responses to substrate topography</h3>
								<p><span class="image right"><img src="img/Bcell_clustering_1_labels.png" width="60%" alt=""/></span>
									A simple physical picture is obtained by considering the cell as an idealized stiffness-measuring device:
									during a measurement, the device exerts a local force on the substrate and reads out a local elastic modulus.
									How this device is realized by the structural and biochemical readout available to the cell and how it affects
									cell remains to be determined. While efforts have been made in this direction in various cellular contexts,
									how stiffness influences immune cell function has largely been unexplored until recently. We demonstrated that
									T cells are mechanosensitive, altering their signaling dynamics and forces
									generated in response to substrate stiffness [4]. We found that forces increase monotonically with stiffness
									of the external surface indicating that cells reorganize their cytoskeletal machinery adaptively. A model
									from active matter theory is able to explain this behavior based on cellular adaptation to substrate rigidity
									resulting from interplay between passive elasticity and active contractility.</p>

									<p><span class="image right"><img src="img/Bcell_clustering_1_labels.png" width="60%" alt=""/></span>The
										mechanisms underlying the origins of mechanosensitivity are not known, but likely involve adaptive
										changes in the organization and mechanical properties of the actin cytoskeleton with subsequent changes
										in cell compliance and shape. In support of this view, we showed that tumor associated fibroblast cells
										are sensitive to the stiffness of the substrate generating larger forces on stiffer substrates [8].
										This adaptive behavior required an actin-crosslinking protein (palladin) for the assembly of contractile
										myosin bundles for force generation. As palladin is implicated in some forms of pancreatic cancer, our
										studies suggest that actin crosslinkers and myosin motors tune the “mechanical state” of the cell and
										alterations in their function leads to aberrant behavior such as in cancer metastases. These results
										highlight the importance of cell mechanical properties in modulating cell function and re-iterate the
										connection between cytoskeletal network structure (regulated by actin crosslinkers) and dynamics in cells.
										Our results shed light on how stiffness of the tumor microenvironment regulates cancer cell shape and
										dynamics and will serve to constrain theoretical models of actin network assembly and force generation in cells.</p>
										<p>Lab Members Involved: <a href="lab_folks.html#king">Altug Ozcelikkale</a>,  <a href="lab_folks.html#king">Kaustubh Wagh</a><br>
										unding: <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1563355&HistoricalAwards=false">NSF Biomechanics and Mechanobiology</a>
							</div>
							<div id="div7" class="content" style="display:none;">
								<h3>Cytoskeletal control of gene expression</h3>
								<p>The molecular and biophysical mechanisms by which mechanics influence cell fate and behavior are
								open questions in the field.  Beyond the modulation of cellular processes such as cell shape,
								migration, signaling, cytoskeletal organization and dynamics (which has been our focus), the
								ultimate consequence of mechanosensing is the modulation of gene expression and changes in cell fate.
								Forces generated by the cytoskeleton as the cell adapts to mechanical stimuli are (thought to be)
								transmitted to the nucleus and chromatin via physical links on the nuclear envelope. How cytoskeletal
								forces affect nuclear dynamics and chromatin accessibility to transcription factors to drive gene
								expression are not well understood. Also, most insights about gene regulation by mechanical inputs
								have been gained from bulk analysis of gene expression patterns, which makes it difficult to link
								how forces generated by individual cells affect chromatin accessibility and transcription factor
								dynamics driving gene regulation. We have recently established collaboration with Dr. Gordon Hager
								(NIH) to pursue these questions and bring novel insight by combining our expertise in the fields of
								biophysics of force generation and biology of the regulation of gene expression.</p>

								<div class="box alt">
									<div class="row 50% uniform">
										<div class="4u"><span class="image fit"><img src="img/nucleus_stress_xmission_soft.png" alt="" /></span></div>
										<div class="4u$"><span class="image fit"><img src="img/nucleus_stress_xmission_stiff.png" alt="" /></span></div>
									</div>
								</div>

								<p>Lab Members Involved: <a href="lab_folks.html#king">David Garcia</a><br>
								Collaborators: <a href="https://ccr.cancer.gov/Laboratory-of-Receptor-Biology-and-Gene-Expression">Hager Lab, NIH/NCI</a><br>
								Funding: <a href="https://cancertechnology.umd.edu/">NCI/UMD Cancer Partnership</a></p>
							</div>
							<div id="div8" class="content" style="display:none;">
								<h3>Force transmission in actin networks</h3>
								<p><span class="image right"><img src="img/Palladin_increasing_conc_noalpha.png" width="60%" alt=""/></span>Actin-binding proteins crosslink actin filaments into viscoelastic gels and organize them into
									diverse cellular structures such as bundles or meshworks. Small changes in binding energies can
									cause the system to switch from network dominated to bundle dominated regimes. These crosslinkers
									determine the length and orientational order of filaments and the inter-filament spacing, and
									thereby control the response of the network to external forces. Palladin is a newly identified
									actin crosslinking protein that is important in organizing cellular actin networks and critical
									for cell movements and morphology. In addition to binding actin palladin also binds other actin
									associated proteins such as alpha-actin and several signaling intermediaries.</p>

									<p><span class="image right"><img src="img/Palladin_increasing_conc_noalpha.png" width="60%" alt=""/></span>We have
										studied how palladin, by its ability to cross-link actin and its interaction with another actin cross-linker,
										alpha-actinin, determines the structure and mechanical properties of actin networks. Using a reconstituted
										system of composite actin networks crosslinked with palladin and alpha-actinin, we have found that these
										mutually interacting crosslinkers synergistically modulate network viscoelastic properties [Grooman et al.,
										PLoS One, 2012].</p>

									<p><span class="image left"><img src="img/magnetic_trap_actin1_noalpha.png" width="60%" alt=""/></span>Cells
										encounter diverse forces as they move through extracellular matrix or tissues. To study how forces are
										transmitted through cells and groups of cells, we use magnetic tweezers to directly apply external forces
										to cells and measure their response by quantifying the strain changes in actin bundles called stress fibers.
										Local application of force leads to a global cellular response, which depends on various physical parameters
										of the applied force. </p>
							 		<p>Lab Members Involved: <a href="lab_folks.html#brian">Brian Grooman</a>, <a href="lab_folks.html#sharon">Shirong Zhang</a><br>
							 			 Collaborators: <a href="http://www.med.unc.edu/cellbiophysio/faculty/otey/">Otey
									 	 Lab, UNC Chapel Hill </a><br>
							 		 	 Grant/Fellowship: NSF-MCB</p>
							</div>
						</div>
					</div>
				</div>
			</section>
		<!-- Scripts -->
			<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
			<script src="js/jquery.fancybox.min.js"></script>
			<script src="js/skel.min.js"></script>
			<script src="js/util.js"></script>
			<script src="js/main.js"></script>
			<script type="text/javascript">
		  	$(document).ready(function() {
		    	$(".fancybox").fancybox();
		  	});
			</script>
			<script>
			$(function() {
				$('#showdiv1').click(function() {
					$('div[id^=div]').hide();
					$('#div1').show();
				});
				$('#showdiv2').click(function() {
					$('div[id^=div]').hide();
					$('#div2').show();
				});

				$('#showdiv3').click(function() {
					$('div[id^=div]').hide();
					$('#div3').show();
				});

				$('#showdiv4').click(function() {
					$('div[id^=div]').hide();
					$('#div4').show();
				});
				$('#showdiv5').click(function() {
					$('div[id^=div]').hide();
					$('#div5').show();
				});
				$('#showdiv6').click(function() {
					$('div[id^=div]').hide();
					$('#div6').show();
				});
				$('#showdiv7').click(function() {
					$('div[id^=div]').hide();
					$('#div7').show();
				});
				$('#showdiv8').click(function() {
					$('div[id^=div]').hide();
					$('#div8').show();
				});
			})
			</script>
			<script>
			$(function () {
    		$("#collapse li").children('ul').hide();
    		$("#collapse li").on('click', function (event) {
        		$(this).children('ul').stop().slideToggle(100);
        		$(this).toggleClass("open");
        		event.stopPropagation();
    		});
			});
			</script>
